x-pack/plugin/inference/src/internalClusterTest/java/org/elasticsearch/xpack/inference/queries/SemanticQueryBuilderMultiClusterIT.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.queries;
+
+import org.elasticsearch.action.search.SearchRequestBuilder;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.test.AbstractMultiClustersTestCase;
+import org.elasticsearch.xpack.inference.InferencePlugin;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
+import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertResponse;
+import static org.hamcrest.Matchers.equalTo;
+
+public class SemanticQueryBuilderMultiClusterIT extends AbstractMultiClustersTestCase {
```

Comment:
This looks to be a very expensive query to start multiple clusters just to test that this query fails on css?

Do you think we can skip this test?

Reply:
Could we add this test to already existing multi cluster integration tests (for example: `CrossClustersQueryIT`?)

Reply:
`CrossClustersQueryIT` seems to be specifically for ESQL. However, I think we could add this test to the general-purpose QA multi-cluster yaml tests: https://github.com/elastic/elasticsearch/tree/a451511e3a127e0c9f2b8c91d67a7d2c4e66c44f/qa/multi-cluster-search/src/test/resources/rest-api-spec/test

WDYT?